app/src/main/java/net/tensory/rxjavatalk/data/BattleFrontFeed.java

```diff
@@ -21,12 +22,21 @@
         SOUTHERN
     }
 
-    private final Front front;
     private final DragonManager dragonManager;
 
+    private BehaviorSubject<Battle> battleSubject = BehaviorSubject.create();
+
     public BattleFrontFeed(Front front, DragonManager dragonManager) {
```

Comment:
I thought keeping all of our magic for the fake feeds local to the BattleFrontFeed would be helpful, since we shouldn't have to show this class during the talk